test/avocado/run-tests

```diff
@@ -134,7 +134,7 @@ def run_avocado_tests(machine, avocado_tests, print_failed=True, env=[]):
 
 def copy_avocado_logs(machine, title):
     if machine and machine.address:
-        dir = "%s-%s.avocado" % (machine.address, title)
+        dir = os.path.join(testinfra.TEST_DIR, "tmp", "%s-%s.avocado" % (machine.address, title))
```

Comment:
Not sure. I guess @jscotka would be the one to ask. I'm just reacting to verify machines with full disks.